src/views/memory.ts

```diff
@@ -153,7 +153,7 @@ export class MemoryViewProvider implements vscode.WebviewViewProvider {
                 case "FetchData": {
                     const type = message.type;
                     if (processKey === '') {
-                        processKey = this.storeLiveProcesses.values().next().value.liveProcess.processKey;
+                        processKey = this.storeLiveProcesses.values().next().value.liveProcess?.processKey;
                     }
                     if (type !== '' && type === "Heap Memory" && processKey !== undefined && processKey !== '') {
```

Comment:
> type !== '' && type === "Heap Memory"

`type !== ''` is obviously unnecessary. A lot of chaos in conditional checks, we might fix that in a separate PR, gracefully checking whether type of processkey is null or empty. @testforstephen